pages/docs/concepts/asyncapi-document/add-server.md

```diff
@@ -0,0 +1,90 @@
+---
+title: Add servers
+weight: 295
+---
+
+A server may represent a message broker handling communication between producers and consumers. It can also represent other things so please first familiarize first with [Server](/docs/concepts/server) concept.
+
+Adding and defining servers is useful because it specifies where and how to connect. The connection facilitates where to send and receive messages.
```

Comment:
```suggestion
Adding and defining servers is important because it specifies how and where connections are made, enabling the sending and receiving of messages.
```